### ISSUE TITLE
Clean up the header UI

### ISSUE THREAD

**Issue by tecimovic** [project-chip/zap#322]
Looking at this section:
![image](https://user-images.githubusercontent.com/7923190/143057884-2070b3dc-7246-45bb-a409-7a2058ec058d.png)

This UI looks a bit weird. We are adding more and more buttons and dropdowns here, and they are all squished.

I would like the following:
   1.) Create a collapsible section right under "Zigbee Cluster Configurator" lable, with label "Global options..."
   2.) By default, that section is collapsed and you just see the twistie that can uncollapse it.
   3.) When you uncollapse it, you see the are which has those 5 options that we have now: "Manufacturer", "Default Response Policy", etc., but each is in its own row, nicely laid out, not squished in the same row.

So basically instead of this super-busy header with 5 controls, we move the controls to a new area, which is collapsible, and does not clog the UI right out of the box.

@jingteng25742 ? @brdandu ? @rerasool ? Any additional ideas around this?

**Comment by tecimovic** [comment]
Some work was done, but I'd like more.....

This takes too much real-estate:
![image](https://user-images.githubusercontent.com/7923190/150845071-5a9075e7-890b-4719-86df-4015ad2817b8.png)

It makes no sense that "Global Options", which is SMALLER font than "Zigbee Cluster Configurator", is taking MORE space vertically. Could we simply add the "Global options" with that open error in a little row to the right of Zigbee Cluster Configurator.

Further more, I don't even like that "Zigbee Cluster Configurator" labe. That whole thing makes no sense to me.... What should we do there?

I do still like the idea of opening this into a fold-out pane. That works ok.
